.github/workflows/static_code_analysis.yaml

```diff
@@ -38,3 +38,10 @@ jobs:
       - name: Check license header and file
         run: |
           python3 ./.devcontainer/license_checker.py
+      - name: Check if openapi.yaml is up to date
```

Comment:
So the goal is that we always check in an `openapi.yaml` file that corresponds to the current API? Can we do that with `pre-commit`?

Reply:
Hm, not sure whether this really fits the pre-commit logic. Typically pre-commit is not interpreting connections between multiple file its only running the checks file by file. But I think in principle we can make that happen somehow. If you like you can add a ticket to the backlog.